cmd/containerd-shim-runhcs-v1/main.go

```diff
@@ -36,6 +37,28 @@ var (
 	idFlag string
 )
 
+func stack() []byte {
```

Comment:
Use `debug.Stack` instead. https://golang.org/pkg/runtime/debug/#Stack

Reply:
Ah, no. `debug.Stack` calls `runtime.Stack` with a large enough buffer -but- doesn't grab all goroutines.

Reply:
Good point, forgot about that.

Reply:
After thinking about it some more, don't we just want the stack for the panicking goroutine? Why do we need stacks for everything?

Reply:
Because I can pretty much guarantee you that at some point we will be very grateful for it - better to have to much info than too little.